backend/src/__tests__/integration.ts

```diff
@@ -67,6 +72,27 @@ describe('Chart App', () => {
       .expect(400);
   });
 
+  it('should create a comment correctly for an existing thread', async () => {
```

Comment:
Added a test case to cover the logic around responding to an existing comment thread